gl.h

```diff
@@ -0,0 +1,575 @@
+#ifndef OBJ_H
+#define OBJ_H
+
+#include "glm/glm.hpp"
+#include "glm/gtx/transform.hpp"
+#include "glm/gtc/matrix_transform.hpp"
+#include "gl/glew.h"
+#include "SDL2/SDL_opengl.h" 
+#include "stdio.h"
+#include "stdlib.h"
+#include <cstring>
+#include <string>
+#include <fstream>
+#include <iostream>
+#include <vector>
+
+// More references 
+#define SCREEN_W 640.0f
+#define SCREEN_H 480.0f
+
+// Abbreviate namespace references 
+using glm::mat4;
```

Comment:
Just curious, can you say "using namespace std" and then only have to set the specific individual glm namespaces?